File: triangle_types.c
#include<stdio.h>
#include<math.h>

int main(){
	double a, b, c;
	scanf("%lf%lf%lf", &a, &b, &c);

	// não forma triangulo
	if (a >= (b + c)) {
		printf("NAO FORMA UM TRIANGULO\n");
	} else if ( pow(a, 2) == (pow(b, 2) + pow(c, 2)) ){
		printf("TRIANGULO RETANGULO\n");
	} else if ( pow(a, 2) > (pow(b, 2) + pow(c, 2)) ){
		printf("TRIANGULO OBTUSANGULO\n");
	}else if( pow(a, 2) < (pow(b, 2) + pow(c, 2)) ){
		printf("TRIANGULO ACUTANGULO\n");
	}
	if (a == b && b == c) {
		printf("TRIANGULO EQUILATERO");
	} else if (a == b || b == c) {
		printf("TRIANGULO ISOSCELES");
	}

	return 0;
}

File: triangle_types.js
var input = require('fs').readFileSync('stdin.txt', 'utf8');
var lines = input.split(' ');

for(var i = 0; i < lines.length; i++) {
	lines[i] = parseFloat(lines[i]);	
}

var [a, b, c] = [...lines];

if (a >= (b+c) || b >= (a+c) || c >= (a+b)) {
	console.log("NAO FORMA UM TRIANGULO");
} else if (a*a == (b*b + c*c) || b*b == (a*a + c*c) || c*c == (a*a + b*b)) {
	console.log("TRIANGULO RETANGULO");
} else if (a*a > (b*b + c*c) || b*b > (a*a + c*c) || c*c > (a*a + b*b)) {
	console.log("TRIANGULO OBTUSANGULO");
} else if (a*a < (b*b + c*c) || b*b < (a*a + c*c) || c*c < (a*a + b*b)) {
	console.log("TRIANGULO ACUTANGULO");
}

if (a === b && b === c) {
	console.log("TRIANGULO EQUILATERO");
}
if ((a === b && a !== c) || (a === c && a !== b) || (b === c && b !== a) ) {
	console.log("TRIANGULO ISOSCELES");
}

